src/Maker/MakeRegistrationForm.php

```diff
@@ -339,14 +339,13 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
                     'from_email' => $this->fromEmailAddress,
                     'from_email_name' => addslashes($this->fromEmailName),
                     'email_getter' => $this->emailGetter,
-                    'authenticator_class_name' => $this->autoLoginAuthenticator ? Str::getShortClassName($this->autoLoginAuthenticator) : null,
-                    'authenticator_full_class_name' => $this->autoLoginAuthenticator,
-                    'firewall_name' => $this->firewallName,
                     'redirect_route_name' => $this->redirectRouteName,
-                    'password_hasher_class_details' => $generator->createClassNameDetails(UserPasswordHasherInterface::class, '\\'),
+                    'password_hasher_class_details' => $hasherDetails = $generator->createClassNameDetails(UserPasswordHasherInterface::class, '\\'),
+                    'password_hasher_variable_name' => sprintf('$%s', lcfirst($hasherDetails->getShortName())),
```

Comment:
Does this stuff still need to be dynamic?

Reply:
Dunno, I thought it was a good idea 2 years ago :D I'll check haha